packages/shared/lib/core/layer-2/utils/getErc20TokenSymbol.ts

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable no-console */
+
+import { get } from 'svelte/store'
+
+import { network } from '@core/network/stores'
+
+import { ContractType } from '../enums'
+
+export async function getErc20TokenSymbol(tokenAddress: string, chainId: number): Promise<string | undefined> {
```

Comment:
do we need an own function for this? this is only used in the ImportErc20Token Popup and there only to display the symbol in the success toast. 
I think we can also use the `getErc20Metadata` function there

Reply:
pushed